src/MatrixClientPeg.js

```diff
@@ -71,7 +72,21 @@ class MatrixClientPeg {
         const opts = utils.deepCopy(this.opts);
         // the react sdk doesn't work without this, so don't allow
         opts.pendingEventOrdering = "detached";
-        this.get().startClient(opts);
+
+        let promise = q();
+        if (this.matrixClient.store instanceof Matrix.IndexedDBStore) {
```

Comment:
Done. https://github.com/matrix-org/matrix-js-sdk/pull/363/commits/c93c8a79b72cc3e95a0c7d1948b156106b8f658d and https://github.com/matrix-org/matrix-react-sdk/pull/687/commits/cca266c62cea505d8e80e5e32cac07bfccc1caf3